spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-postgresql.sql

```diff
@@ -58,3 +58,19 @@ CREATE TABLE INT_METADATA_STORE  (
 	REGION VARCHAR(100) NOT NULL,
 	constraint INT_METADATA_STORE_PK primary key (METADATA_KEY, REGION)
 );
+
+-- This is only needed if using PostgresChannelMessageSubscriber
+-- CREATE FUNCTION INT_CHANNEL_MESSAGE_NOTIFY_FCT()
+-- RETURNS TRIGGER AS
+-- $BODY$
+-- BEGIN
+--     EXECUTE 'NOTIFY int_channel_message_notify, ''' ||  QUOTE_LITERAL(NEW.GROUP_KEY || NEW.REGION) || '''';
```

Comment:
Why haven't you yet flipped as `region | groupKey`?

Reply:
Just an oversight, just fixed it.